conanfile.py

```diff
@@ -40,8 +40,8 @@ def build_requirements(self):
         self.build_requires("gtest/1.14.0")
 
     def requirements(self):
-        self.requires("homestore/[>=6.2,      include_prerelease=True]@oss/master")
-        self.requires("sisl/[>=12.1, include_prerelease=True]@oss/master")
+        self.requires("homestore/[^6.2, include_prerelease=True]@oss/master")
```

Comment:
question: if HS made some api change that will break HO build, previously we only bump minor version, this stays true still, right? e.g. we don't need to bump major, just bump minor for HS.

Reply:
I am not sure...previously we bumped the major version. We discussed in HS meeting but I am not sure a decision has been made.

If that is the case, we need to pin to [^6.2.0]